src/views/activity/activity_item_view.js

```diff
@@ -21,5 +27,35 @@ export default BaseView.extend({
 
   sendSelectionDetails (event) {
     webChannel.sendUrlSelected(this.model.url, 'url');
+  },
+
+  injectFavicon () {
+    // Manually construct the favicon element and inject it into the DOM,
+    // allowing us to handle events directly, since load events don't bubble
+    // and cannot be delegated.
+    this.icon = new Image('img');
+    this.icon.onload = this.calculateIconColor();
+    this.icon.onerror = this.hideErroredFavicon();
```

Comment:
I think it would be cleaner to change the callbacks back to regular functions, then bind them explicitly when they are attached to `this.icon` here inside `injectFavicon`:
```
this.icon.onload = this.calculateIconColor.bind(this);
this.icon.onerror = this.hideErroredFavicon.bind(this);
```
Alternatively, I guess `this.icon.onload = () => { this.calculateIconColor() }` would do the same thing in a more futuristic way.

Whichever one you choose, I think we definitely want to avoid the `return () => {}.bind(this)` idiom. I don't think we should ever need to bind an arrow func.